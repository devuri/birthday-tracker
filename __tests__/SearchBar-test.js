import 'react-native';
import React from 'react';
import SearchBar from '../app/components/SearchBar.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <SearchBar />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
