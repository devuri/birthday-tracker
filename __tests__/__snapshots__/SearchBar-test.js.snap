exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#EAE8E8",
      "borderRadius": 10,
      "marginBottom": 10,
      "marginLeft": 0,
      "marginRight": 0,
      "marginTop": 80,
      "width": 300,
    }
  }>
  <TextInput
    autoCapitalize="none"
    autoCorrect={false}
    onChange={undefined}
    placeholder="Search contacts"
    style={
      Object {
        "color": "#02556D",
        "height": 40,
        "textAlign": "center",
      }
    } />
</View>
`;
